run.go

```diff
@@ -220,21 +221,25 @@ func runff(ff *funcfile, stdin io.Reader, stdout, stderr io.Writer, method strin
 				return fmt.Errorf("error creating http request: %v", err)
 			}
 			req.Header.Set("Content-Type", contentType)
-			req.Header.Set("Fn_deadline", deadlineS)
+
+			req.Header.Set("FN_REQUEST_URL", LocalTestURL)
```

Comment:
Should these by in the same case as here: https://github.com/fnproject/fn/pull/660#issuecomment-356157279 ?

Reply:
Pretty sure that header.Set canonicalises so don't think it matters here - just trying to be consistent with the headers in http.go